lib/primitives/tx.js

```diff
@@ -1960,8 +1958,11 @@ class TX {
         if (controlBlock && controlBlock.length === 0)
           return false;
 
-        // Leaf version 0xc0 (aka Tapscript, see BIP 342)
-        if (controlBlock && ((controlBlock[0] & 0xfe) === 0xc0)) {
+        // Leaf version Tapscript
+        if (
+          controlBlock &&
+          ((controlBlock[0] & 0xfe) === common.LEAF_VERSION_TAPSCRIPT)
+          ) {
```

Comment:
nit: just missing indentaiton here
```suggestion
        if (
            controlBlock &&
            ((controlBlock[0] & 0xfe) === common.LEAF_VERSION_TAPSCRIPT)
          ) {
```